css/style.css

```diff
@@ -992,28 +1034,30 @@ textarea {
   }
 
   #modal-heading {
-    font-size: 28px;
+    font-size: 25px;
+    margin-top: 30px;
   }
 
-  #modal-form .form-item-spacing {
-    flex-direction: column;
+  .position-heading-close img {
+    top: -35px;
```

Comment:
@aromal-qburst Position the close with respect to the modal to avoid negative px values

Reply:
Done